.circleci/config.yml

```diff
@@ -54,7 +54,7 @@ jobs:
       - run:
           name: Deploy Over SSH
           command: |
-            ssh $SSH_USER@$SSH_HOST "bash ~/hetmech-backend/.circleci/deploy.sh"
+            ssh $SSH_USER@$SSH_HOST "bash ~/connectivity-search-backend/.circleci/deploy.sh"
```

Comment:
Actually this should be reverted right since the server still uses `hetmech-backend`

Reply:
Done in https://github.com/greenelab/connectivity-search-backend/pull/72/commits/c05e276c4e49a9d2e2eaa5c0076800dd10142dbe